src/main/java/Ui.java

```diff
@@ -0,0 +1,111 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Ui {
+    private Scanner sc;
+
+    public Ui() {
+        this.sc = new Scanner(System.in);
+    }
+
+    /**
+     * printLine() will print the separator lline from input and output.
```

Comment:
This should be line and not **l**line. Perhaps you should use IntelliJ's code inspection utilities (Analyze → Inspect Code), so any such typos can be caught before they are committed?